src/xGadgetron/pGadgetron/Gadgetron.py

```diff
@@ -1202,6 +1212,10 @@ def backward(self, ad):
             (self.handle, ad.handle)
         check_status(image.handle)
         return image
+    def direct(self, image):
```

Comment:
Why does `CoilImageData` have a method `direct`? This is unintuitive to me.

Reply:
what about `adjoint`? Should it be there?

Reply:
Well, I don't know. To me this is a data object and not an operator. So it should neither have `direct`, `forward` nor `adjoint`. But it could be that I did not see any specific CCP-PETMR point of view on this. @KrisThielemans ?

Reply:
agree with Matthias - this class should have neither direct nor adjoint